<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilmWindow</class>
 <widget class="QWidget" name="FilmWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="1" colspan="3">
      <layout class="QGridLayout" name="jogGrid">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="jobLabel">
         <property name="text">
          <string>Jog Motor</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2" colspan="2">
        <widget class="QDial" name="jogDial">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="minimum">
          <number>-210</number>
         </property>
         <property name="maximum">
          <number>210</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="notchTarget">
          <double>20.000000000000000</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2" rowspan="2">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>250</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QPushButton" name="jogHomeButton">
           <property name="text">
            <string>Home</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="jogEndButton">
           <property name="text">
            <string>End</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QDoubleSpinBox" name="jogSpeedSpin">
           <property name="maximum">
            <double>200.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="QDoubleSpinBox" name="jogDampSpin"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="jogSpeedLabel">
           <property name="text">
            <string>Max Jog Speed</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QLabel" name="jogDampLabel">
           <property name="text">
            <string>Damping (Sec)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="3" rowspan="3">
        <widget class="QListView" name="devButtonList">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>72</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>306</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>50</width>
           <height>250</height>
          </size>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="midLineWidth">
          <number>1</number>
         </property>
         <property name="autoScroll">
          <bool>true</bool>
         </property>
         <property name="autoScrollMargin">
          <number>8</number>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::MoveAction</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0" colspan="4">
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="6" column="1" colspan="4">
      <widget class="QScrollArea" name="visualSA">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>75</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <widget class="QWidget" name="visualSAContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>784</width>
          <height>178</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="Line" name="line_2">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>300</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="ControlButHoriz">
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="camControlCheckBox">
           <property name="text">
            <string>Camera Control</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="camSetBut">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Film Length</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="FilmLenGrid">
         <item row="1" column="0">
          <widget class="QSpinBox" name="filmHHSpin"/>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="filmMMSpin"/>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="filmSSSpin"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>HH</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>MM</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>SS</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Real Time</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="RealLenGrid">
         <item row="1" column="0">
          <widget class="QSpinBox" name="realHHSpin"/>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="realMMSpin"/>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="realSSSpin"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>HH</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>MM</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>SS</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="TimeDispGrid">
         <item row="1" column="1">
          <widget class="QLCDNumber" name="filmHHLCD"/>
         </item>
         <item row="1" column="2">
          <widget class="QLCDNumber" name="filmMMLCD"/>
         </item>
         <item row="1" column="3">
          <widget class="QLCDNumber" name="filmSSLCD"/>
         </item>
         <item row="3" column="1">
          <widget class="QLCDNumber" name="runHHLCD"/>
         </item>
         <item row="3" column="2">
          <widget class="QLCDNumber" name="runMMLCD"/>
         </item>
         <item row="3" column="3">
          <widget class="QLCDNumber" name="runSSLCD"/>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Film Time</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="3">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Run Time</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLCDNumber" name="curFrameLCD"/>
         </item>
         <item row="5" column="3">
          <widget class="QLCDNumber" name="totFrameLCD"/>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Current Frame</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLabel" name="label_12">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Total Frames</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="6">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>200</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="seekbuttonhoriztonallayout">
         <item>
          <widget class="QPushButton" name="rewindButton">
           <property name="text">
            <string>|&lt;&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="frameRwdButton">
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="playButton">
           <property name="text">
            <string>Play</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopButton">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="frameFwdButton">
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="forwardButton">
           <property name="text">
            <string>&gt;&gt;|</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>180</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1" colspan="4">
      <spacer name="widthHSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>792</width>
         <height>1</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1" colspan="4">
      <layout class="QVBoxLayout" name="tapeVLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
